src/libraries/System.Private.Xml.Linq/tests/TrimmingTests/XElementCtor.cs

```diff
@@ -0,0 +1,177 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.Serialization;
+
+public class Program
```

Comment:
In my test I added an xml summary comment to the program that explained what this was testing. I think it makes since to add a sentence or two explaining what this is testing.